app/models/host_report.rb

```diff
@@ -56,6 +57,19 @@ def self.search_by_keyword(_key, operator, value)
     }
   end
 
+  def self.summarise(_time = 1.day.ago, *hosts, format)
+    list = {}
+    raise ::Foreman::Exception, N_("invalid host list") unless hosts
```

Comment:
Please see the wiki page about how to use Foreman::Exception.

Reply:
I know how :) but the rubocop failed because of that, it mention to change it to that from some reason.